parking/backend/db/sql_constants.py

```diff
@@ -0,0 +1,61 @@
+# flake8: noqa
+
+PARKINGLOTS_CREATE_TABLE = """
+CREATE TABLE ParkingLots (
+	id 	            serial PRIMARY KEY,
+	name 	        text NOT NULL,
+	capacity        integer NOT NULL,
+	lat 	        float NOT NULL,
+	long	        float NOT NULL,
+	price		    float NOT NULL,
+	num_available   integer NOT NULL,
+	num_allocated 	integer NOT NULL
+);
+"""
+
+ALLOCATIONS_CREATE_TABLE = """
+CREATE TABLE Allocations (
+    user_id text NOT NULL,
+    park_id integer NOT NULL,
+
+    PRIMARY KEY(user_id),
```

Comment:
Can the column be marked as a primary key in one line, as you did for ParkingLots.id?